lib/core/i18n/i18n.utils.ts

```diff
@@ -9,7 +12,10 @@ import { Maybe } from "@trpc/server";
 import { i18n } from "../../../next-i18next.config";
 
 export function getLocaleFromHeaders(req: IncomingMessage): string {
-  const preferredLocale = parser.pick(i18n.locales, req.headers["accept-language"]) as Maybe<string>;
+  const preferredLocale = parser.pick(
+    i18n.locales,
+    req.headers["accept-language"] as string
+  ) as Maybe<string>;
```

Comment:
I think this will conflict with my PR: https://github.com/calendso/calendso/pull/988/files#r731431175

Reply:
Not necessarily a problem, we can fix the conflict later depending which PR gets merged first.